__tests__/app.test.js

```diff
@@ -159,3 +159,38 @@ describe("Users", () => {
     });
   });
 });
+describe("Comments", () => {
+  describe("getCommentsById", () => {
+    test("status: 200 and resonds with an array of comments objects relating to the article_id", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { article } }) => {
```

Comment:
This should be being sent back on a key of comments please! It's an array of comments, not an array of articles.